### ISSUE TITLE
Chart does not fit parent

### ISSUE THREAD

**Issue by galvakojis** [swimlane/ngx-charts#1142]
**I'm submitting a ...**  (check one with "x")

- [x] bug report - search github for a similar issue or PR before submitting
- [ ] feature request
- [ ] support request - use StackOverflow (add the `ngx-charts` tag) or the gitter chat for support questions

**Current behavior**

strange behavior of fitting parent object, content does't fit
https://stackblitz.com/edit/angular-gridster-charts
after a press share it somehow resizes and fits into parent content
**Expected behavior**

should be normal on load
**Reproduction of the problem**
<!-- If the current behavior is a bug or you can illustrate your feature request better with an example, please provide the steps to reproduce and if possible a minimal demo of the problem https://stackblitz.com/edit/angular-gridster-charts

**Comment by marjan-georgiev** [comment]
Correct, the length of the label with `maxLabelLength` doesn't get taken into account.
